### PR TITLE
User/xianz/winmldb

### PR DESCRIPTION
**Pull request by zhangxiang1993**

### CODE REVIEW

**Comment by zhangxiang1993** [comment]
image

File: Tools/WinMLDashboard/README.md [hunk 1]
```diff
@@ -28,10 +28,18 @@ The **Edit/View** button switches from Edit mode to View-only mode and vice vers
...
+The **Convert** tab (see snip below) helps convert models from several different frameworks (as listed above) to ONNX format by [WinMLTools](https://pypi.org/project/winmltools/). Also you can select the ONNX version to convert to and select to quantize on which OS version. 
 
 In order to do the conversion, the tool installs a separate Python environment and a set of converter tools. This helps alleviate one of the big pain points of a typical developer - installing the right Python environment and tool chain for conversion and validating the model. 
 
+WinMLTools provides quantization tool to reduce the memory footprint of the model.
+
+For **quantization**:
+| | |
+|-|-|
+|**Type1**|Do not use dequantize_linear option, the converted model can only be run in Win10 released before September 2018.|
```
Comment (kumraj) [inline Tools/WinMLDashboard/README.md]: This should be a table

File: Tools/WinMLDashboard/README.md [hunk 2]
```diff
@@ -28,10 +28,18 @@ The **Edit/View** button switches from Edit mode to View-only mode and vice vers
 
 ## Converting Models
 
-The **Convert** tab (see snip below) helps convert models from several different frameworks (as listed above) to ONNX format. 
+The **Convert** tab (see snip below) helps convert models from several different frameworks (as listed above) to ONNX format by [WinMLTools](https://pypi.org/project/winmltools/). Also you can select the ONNX version to convert to and select to quantize on which OS version.
```
Comment (kumraj) [inline Tools/WinMLDashboard/README.md]: ONNX version to convert to and apply Quantization.

File: Tools/WinMLDashboard/src/view/convert/View.tsx [hunk 3]
```diff
@@ -244,7 +244,11 @@ class ConvertView extends React.Component<IComponentProperties, IComponentState>
                         onChange={this.setONNXVersion}
                         options={ONNXVersionOptions}
                     />
-                    <label className='label-center-align'>Quantization: </label>
+                    <label className='label-center-align'>
+                        Quantization
+                        <img onClick={this.openReadMe} src="https://shots.jotform.com/kade/Screenshots/blue_question_mark.png"
+                        title="WinMLTools provides quantization tool to reduce the memory footprint of the model." height="18px"/>:
```
Comment (kumraj) [inline Tools/WinMLDashboard/src/view/convert/View.tsx]: replace tool with feature

File: Tools/WinMLDashboard/README.md (hunk 2 above)
Comment (zhangxiang1993) [reply Tools/WinMLDashboard/README.md]: changed

File: Tools/WinMLDashboard/src/view/convert/View.tsx (hunk 3 above)
Comment (zhangxiang1993) [reply Tools/WinMLDashboard/src/view/convert/View.tsx]: changed.

File: Tools/WinMLDashboard/README.md (hunk 1 above)
Comment (zhangxiang1993) [reply Tools/WinMLDashboard/README.md]: changed

Comment (zhangxiang1993) [reply Tools/WinMLDashboard/README.md]: And it is a MarkDown Table.

**Pull request merged**